### PR TITLE
gNALI 1.0.1

### PR DESCRIPTION
**Pull request by xialiu71**
Update gNALI version to 1.0.1 and update tests

### CODE REVIEW

**Comment by kbessonov1984** [comment]
@xialiu71 a couple of suggestions you can add  to the recipe such as 
* Add `on ${test_genes.element_identifier}` to the 3 outputs name strings so users know on which input did gNaLi was run on when Galaxy history is full of items
* Move version code to `macros.xml`
inside the `gnali.xml` add
```
<macros>
    <import>macros.xml</import>
  </macros>  
  <expand macro="requirements" />
```
Add `macros.xml` file
```

<macros>
  <token name="@VERSION@">1.0.0</token>
  <xml name="requirements">
  <requirements>
  <requirement type="package" version ="@VERSION@">gnali</requirement>
  </requirements>
  </xml>
</macros>
```
Hope this is useful

**Comment by kbessonov1984** [comment]
@xialiu71 For faster local testing you can try this docker image with Galaxy installed (https://hub.docker.com/repository/docker/kbessonov/planemo). I use it all the time as Travis CI is very slow these days. If it passes your local `planemo` tests (lint and test commands) then it should pass here.

**Comment by xialiu71** [comment]
Thanks for your suggestion, I've implemented it in f6e4af4. And thanks for the Docker suggestion as well!

**Comment by kbessonov1984** [comment]
Great! Looks like the issue is that the files produced in the run are different from the test ones. Try to run tests manually on a Galaxy instance (e.g. in docker using the `planemo` image above) and copy them into a test folder after making sure they look OK.  One of the tests seems to miss the 'HGNC_Symbol' column causing `diff` to fail

```
Output file did not contain expected text 'HGNC_Symbol' (output '')
History item  different than expected, difference (using diff):
( /tmp/tmpd44p9wznNonessential_Host_Genes_Detailed.txt v. /tmp/tmpf_h76emvNonessential_Host_Genes_Detailed.txt )
```

**Comment by xialiu71** [comment]
Thanks, I've been running the tests locally and I'm unable to reproduce the Travis CI errors. I'm still looking into it

File: tools/gnali/gnali.xml [hunk 1]
```diff
@@ -1,109 +1,135 @@
...
+        #if $database_info.predefined_filters != "None":
+            --predefined_filters '$database_info.predefined_filters'
+        #end if
+        #if $additional_filters != "None":
+            --additional_filters 
+            #for $filt in $additional_filters
+                '$filt.filter' 
+            #end for
+        #end if
+        $vcf_output
```
Comment (bgruening) [inline tools/gnali/gnali.xml:18]: please single quote

File: tools/gnali/gnali.xml [hunk 2]
```diff
@@ -1,109 +1,135 @@
...
+        $pop_freqs
```
Comment (bgruening) [inline tools/gnali/gnali.xml:19]: this one as well

File: tools/gnali/gnali.xml [hunk 3]
```diff
@@ -1,109 +1,135 @@
...
     ]]></command>
     <inputs>
         <param type="data" name="test_genes" label="Test genes" format="txt" help="Specify a list of genes as HGNC symbols, separated by newline characters" />
-        <param type="select" name="database" label="Database" format="txt" help="Database to query" >
-            <option value="gnomad2.1.1" selected="true">gnomAD2.1.1 (GRCh37/hg19)</option>
+        <param name="vcf_output" type="select" label="VCF output" help="Generate vcf file for filtered variants" >
```
Comment (bgruening) [inline tools/gnali/gnali.xml:23]: boolean?

File: tools/gnali/macros.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<macros>
+  <token name="@VERSION@">1.0.0</token>
```
Comment (bgruening) [inline tools/gnali/macros.xml:2]: indentation?

**Review by bgruening — COMMENTED** [review]
Cool, thanks @xialiu71

File: tools/gnali/gnali.xml (hunk 1 above)
Comment (xialiu71) [reply tools/gnali/gnali.xml]: Fixed in 66a3247

File: tools/gnali/gnali.xml (hunk 2 above)
Comment (xialiu71) [reply tools/gnali/gnali.xml]: Fixed in 66a3247

File: tools/gnali/gnali.xml (hunk 3 above)
Comment (xialiu71) [reply tools/gnali/gnali.xml]: Updated in 66a3247

File: tools/gnali/macros.xml (hunk 4 above)
Comment (xialiu71) [reply tools/gnali/macros.xml]: Fixed in 66a3247

**Review by ericenns — APPROVED** [review]
👍

**Review by ericenns — APPROVED** [review]
👍

**Pull request merged**